internal/debug/flags.go

```diff
@@ -90,7 +95,7 @@ var (
 
 // Flags holds all command-line flags required for debugging.
 var Flags = []cli.Flag{
-	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag,
+	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag, debugfileFlag,
```

Comment:
Is the debugFileFlag only for SWAP related debugging, or for the whole Swarm?

Reply:
This is actually for whole Swarm including swap logger as well.

We were discussing with Marcelo the possibility of splitting the outputs.
But we agreed it makes more sense if they are unified.

The swap logger already defines its own prefix for searching all the related swap events.

I would appreciate everybody's advice on this matter, as it would affect the whole swap implementation.
It's only for debug but still something to consider.

On the flip side, if it's separated I would be hard to debug the log in case of a problem between swap and the rest of the code.

Reply:
If we want to change the global debug file handling, then we need to discuss this with core. I think they have precise understanding of how they want to deal with this. I personally don't think we need to change the global debug file behavior.

Reply:
if we are not changing the entire global debug file behavior (which i can certainly agree on) then let's make this parameter/flag explicit; in the sense that it should be clear it is _only_ for swap logs